wire-ios-data-model/Support/Sources/MLS/MLSGroupID+random.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireDataModel
+
+extension MLSGroupID {
+
+    public static func random() -> MLSGroupID {
+        .init(Data.random(byteCount: 32))
```

Comment:
Thank you to extract it, maybe better use a constant here?

Reply:
This seems to be just a random number, since I am not sure and I just moved this code to another file, I prefer to keep it as is.

Reply:
I understand, didn't think of another file but more like:

```swift
extension MLSGroupID {
    private enum Constant {
        static let byteCount: UInt = 32
    }

    public static func random() -> MLSGroupID {
        .init(Data.random(byteCount: Constant.byteCount))
    }
}
```

It's up to you 🤓